docker-compose.yml

```diff
@@ -29,7 +29,6 @@ x-server: &base-server-config
     AWS_S3_SECRET_ACCESS_KEY: ${AWS_S3_SECRET_ACCESS_KEY:-}
     AWS_S3_BUCKET_NAME: ${AWS_S3_BUCKET_NAME}
     AWS_S3_REGION_NAME: ${AWS_S3_REGION_NAME}
-    USE_S3: ${USE_S3:-}
```

Comment:
@k9845  Why did you remove this one? I think this is in use.

Reply:
Defined twice.